core/WFCORE-4314_enchance_keystore_commands_alias.adoc

```diff
@@ -0,0 +1,337 @@
+= Enchance read-alias and read-aliases keystore commands
+:author:            Bartosz Baranowski
+:email:             bbaranow@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+ Elytron keystore expose `read` commands to access content:
+ * read-alias
+ * read-aliases
+
+ There are two problems with those commands. The `read-alias` command is too verbose by default. When alias is displayed after read command, output include hex format of public-key and encoded fields. This is usually not something required if one inspects key and it clouds output with dump of unreadable string of hex digits.
+ Second problem is that read-aliases is in fact "read-aliases-names". In order to read content of keystore one has to read alias names with `read-aliases` and issue `read-alias` for each name. To make it more user friendly, simple changes are required:
+ * VERBOSE - new parameter for both commands. Default value would be `false`. This parameter control inclusion of bulk hex fields of key(public-key and encoded)
+ * RECURSIVE - new parameter for `read-aliases`. Default value `false`. If set otherwise, `read-aliases` will read recurse into keys, displaying more than names.
+
+CAUTION: default value of VERBOSE parameter, in current implementation is `true`. Explicity definition of parameter with default value of `false` is a change in behavior.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFCORE-4314
+
+=== Related Issues
+
+=== Dev Contacts
+
+* mailto:mailto:{email}[{author}]
+* mailto:darran.lofthouse@redhat.com[Darran Lofthouse]
+* mailto:fjuma@redhat.com[Farah Juma]
+* mailto:vobratil@redhat.com[Vilem Obratil]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+[ ] Engineering
```

Comment:
Have you checked with Vilem who will be handling the test plan for this?

Reply:
The RFE has meanwhile moved up in rank, so I suppose that this one is up to me.